### PR TITLE
Allow users to use FAWE & WG hook

### PR DESCRIPTION
**Pull request by IronApollo**
### Description
Since FAWE-WG (our old fork of WorldGuard) has been obsolete for quite a while now, there is no need to continue internally preventing users with FAWE from using the WorldGuard hook. FAWE continues to make strides towards complete API compatibility with upstream WorldEdit, and in turn, WorldGuard should -- and does -- work fine with FAWE.

---
**Target Minecraft Versions:** any
**Requirements:** [FastAsyncWorldEdit](https://github.com/IntellectualSites/FastAsyncWorldEdit)
**Related Issues:** #2674

### CODE REVIEW

**Review by ShaneBeee — APPROVED** [review]
I dont really know much about FAWE + WE/WG, so I dont really know if this fixes this issue or not, but going to take your work for it, and hope it was tested thoroughly.
Im going to approve, but also wait for other members of the team to give feedback and/or approve before merging this.

**Review by Romitou — APPROVED** [review]
Everything works fine according to my personal tests.
`WorldEdit version 1.16-341;68bd411`
`WorldGuard version 7.0.3;5407315`
`FastAsyncWorldEdit version 1.16.2`
`Paper-182 API version 1.16.2-R0.1-SNAPSHOT`

**Pull request merged**
